tests/unittests/Foundation/NSPointerFunctionsTests.mm

```diff
@@ -80,7 +80,8 @@ NSUInteger NSPointerFunctionsTestStructSizeFunction(const void* item) {
     const void* ptr3 = (void*)((long)ptr1 + 1);
 
     testEqualFunction(functions, ptr1, ptr2, ptr3);
-    testHashFunction(functions, ptr1, ptr2, ptr3);
+    // Disabling hash function check here as performing hash on opaque values cannot be accuratly matched with the reference platform.
+    // testHashFunction(functions, ptr1, ptr2, ptr3);
```

Comment:
I would generally agree, but we can't assume that people are only putting allocated memory into these things. There's stuff like function pointers from different modules (different load addresses), section/segment headers, etc.

The problem is that it doesn't even make sense to "hash" pointers. They're already 32-64 full bits of entropy.

---
In reply to: [73764838](https://github.com/Microsoft/WinObjC/pull/833#discussion_r73764838) [](ancestors = 73764838,73764075,73762989)